doc/examples/README.md

```diff
@@ -2,6 +2,8 @@
 
 This folder contains a collection of more specific examples using flame, specially to showcase some more advanced functionality.
 
-To run this examples after cloning, first run `flutter create .` inside to generate the necessary files, as per instructed by the other flutter commands.
+To run this examples after cloning, first run `flutter create .` inside the desired example folder to generate the necessary files, as per instructed by the other flutter commands.
```

Comment:
Not your typo, but would be nice to get fixed in this PR anyways. :)
Should be "To run an example" or "these examples"

Reply:
Correct!

Reply:
Thanks!